src/nunit.analyzers/ConstActualValueUsage/ConstActualValueUsageAnalyzer.cs

```diff
@@ -24,21 +27,90 @@ public class ConstActualValueUsageAnalyzer : BaseAssertionAnalyzer
         protected override void AnalyzeAssertInvocation(SyntaxNodeAnalysisContext context,
             InvocationExpressionSyntax assertExpression, IMethodSymbol methodSymbol)
         {
+            bool IsLiteralExpression(ExpressionSyntax expression)
```

Comment:
I think we should have at least one test per case in this method - so that we don't introduce regressions later on.

Reply:
There are use cases in `AnalyzeWhenLiteralArgumentIsProvidedForAssertThat`: `-1` triggers `PrefixUnaryExpressionSyntax` and the `(2 + 3) * 1024` triggers all the others:  `ParenthesizedExpressionSyntax`, `BinaryExpressionSyntax`, `LiteralExpressionSyntax` Would you prefer these to be in separate test methods?

Reply:
My fault, I overlooked those (so much for reviewing the tests one day and the prod code the next). I think it is fine as it is.